source/includes/fundamentals/code-snippets/srv.go

```diff
@@ -14,6 +14,7 @@ const uri = "<connection string>"
 
 func main() {
 
+	// Use the ServerAPI() method to set the Stable API version to 1
```

Comment:
Suggestion:

Sorry for missing this -- shouldn't it be the SetServerAPIOptions method that actually sets the Stable API version? I think options.ServerAPI() returns a ServerAPIOptions pointer.

```suggestion
	// Use the SetServerAPIOptions() method to set the Stable API version to 1
```